incubator/command-dump/src/main/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommand.java

```diff
@@ -198,13 +223,37 @@ public final class ZillaDumpCommand extends ZillaCommand
 
     public ZillaDumpCommand()
     {
-        this.patchBuffer = new UnsafeBuffer(ByteBuffer.allocate(BUFFER_SLOT_CAPACITY));
-        this.writeBuffer = new UnsafeBuffer(ByteBuffer.allocate(BUFFER_SLOT_CAPACITY));
+        this.patchBuffer = new UnsafeBuffer(ByteBuffer.allocate(PATCH_BUFFER_SLOT_CAPACITY));
+        this.writeBuffer = new UnsafeBuffer(ByteBuffer.allocate(WRITE_BUFFER_SLOT_CAPACITY));
     }
 
     @Override
     public void run()
     {
+        if (pluginDirectory != null)
+        {
+            try
+            {
+                InputStream is = getClass().getResourceAsStream("/wireshark/zilla.lua");
```

Comment:
Please move `zilla.lua` so it is in a subdirectory matching the package of this class, then this becomes `getClass().getResourceAsStream("zilla.lua")`.

i.e. `src/main/resources/io/aklivity/zilla/runtime/command/dump/internal/airline/zila.lua`